langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -341,6 +496,11 @@ public isolated function fromBase64(string str) returns byte[]|error = @java:Met
 # The result will contain only characters  `0..9`, `a..f`.
 # There will be no whitespace in the returned string.
 #
+# ```
+# byte[] byteArray = [170, 171, 207, 204, 173, 175, 205, 52, 26, 75, 223, 171, 205, 137, 18, 223];
+# byteArray.toBase16() => aaabcfccadafcd341a4bdfabcd8912df
```

Comment:
Can't do `[170, 171, 207, 204, 173, 175, 205, 52, 26, 75, 223, 171, 205, 137, 18, 223].toBase16()` due to https://github.com/ballerina-platform/ballerina-lang/issues/38997